CMakeLists.txt

```diff
@@ -194,7 +203,9 @@ if (${RUN_TESTS})
 endif()
 
 add_subdirectory(framework)
-add_subdirectory(layer)
+if(!BUILD_STATIC)
```

Comment:
This seems incorrect.  The `layer` directory was built before if `BUILD_STATIC` was off.  Now it's not?